src/core/Spec.js

```diff
@@ -78,11 +78,11 @@ getJasmineRequireObj().Spec = function(j$) {
   Spec.prototype.onException = function onException(e) {
     if (Spec.isPendingSpecException(e)) {
       this.pend(extractCustomPendingMessage(e));
-      return;
+      return true;
     }
 
     if (e instanceof j$.errors.ExpectationFailed) {
-      return;
+      return false;
```

Comment:
I think that a spec should probably always return `true` to denote it handled any exception. The only case I can think of where we might want it to propagate up the suite chain would be a disabled spec.